src/app/src/utils/canvas.js

```diff
@@ -6,12 +6,80 @@ export const combineCanvases = (canvasList, direction) => {
     }
 };
 
+function generateWrappedText(context, text, width, font) {
+    context.font = font;
+
+    var x = 20;
+    var y = 50;
+    var maxWidth = width - x * 2;
+    var lineHeight = 50;
```

Comment:
How did you arrive at these magic numbers ? Do consider adding inline comments to explain and help our future selves